aldryn_config.py

```diff
@@ -5,31 +5,61 @@
 
 
 class Form(forms.BaseForm):
+
     enable_debug_toolbar = forms.CheckboxField(
         'Enable Django Debug Toolbar',
         required=False,
         initial=True,
     )
 
+    # The generally recommended position for the DebugToolbarMiddleware is immediately after the
+    # GZipMiddleware. See:
+    #
+    # https://django-debug-toolbar.readthedocs.io/en/stable/installation.html#middleware
+    # https://django-debug-toolbar.readthedocs.io/en/stable/tips.html#middleware-isn-t-working-correctly
+
+    POSITIONS = [
+        ["top", "As close to the top as possible (generally recommended)"],
+        ["end", "As close to the end as possible"],
+
+    ]
+
+    toolbar_middleware_position = forms.SelectField(
+        'DebugToolbarMiddleware position',
+        help_text="Position in the middleware list",
```

Comment:
Please consistently use single quotes (except where the string contains a single quote itself). Also below.